include/odp/api/spec/packet_io_types.h

```diff
@@ -257,7 +257,7 @@ typedef struct odp_pktin_queue_param_t {
 	  * More than one input queues require flow hashing configured.
 	  * The maximum value is defined by pktio capability 'max_input_queues'.
 	  * Queue type is defined by the input mode. The default value is 1. */
-	unsigned int num_queues;
+	uint32_t num_queues;
```

Comment:
`doc/users-guide/users-guide-pktio.adoc` has still code fragments with the old `unsigned int` types.

Reply:
User guide includes many copies of API structures. Agreed to minimize amount of copy-paste code in user guide in another PR.